smoke-test/src/test/java/org/opennms/smoketest/minion/DetectorsCommandTest.java

```diff
@@ -0,0 +1,158 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ * <p>
```

Comment:
License header shouldn't contain <p> tags. See https://github.com/OpenNMS/opennms/blob/54c153ed33867efcf8bb9eacbad75345246f02f7/smoke-test/src/test/java/org/opennms/smoketest/minion/SyslogTest.java#L1 for a good example.